e2e/features/login.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Login
+
+  Scenario: User navigates to register
+    Given the user is on the login page
+    And the user clicks on the "Még nincs fiókja?" button
```

Comment:
The text is: "Nincs még fiókja?"
```suggestion
    And the user clicks on the "Nincs még fiókja?" button
```

Reply:
Thanks, would not understand what is going on.